Source/Reporting/Web/src/app/Management/DataCollectors/edit/edit.ts

```diff
@@ -22,13 +22,15 @@ import * as L from 'leaflet';
     templateUrl: './edit.html',
     styleUrls: ['./edit.scss']
 })
-export class Edit implements OnInit {
+export class Edit implements OnInit, AfterViewInit {
     queryCoordinator: QueryCoordinator;
     commandCoordinator: CommandCoordinator;
 
     error = false;
     dataCollector: DataCollector;
     phoneNumberString = '';
+    lat = 9;
```

Comment:
Preferebly you want to declare variables private or public, so that we can mange complexity. Ideally it is something that should have more focus forward. Its just a question of code quality :)

Reply:
Great that you got my back :)